ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -20,6 +20,7 @@ nginx_sites:
    location2:
     name: "/static"
     alias: "{{ nginx_static_home }}"
+    add_header: "Access-Control-Allow-Origin *;"
```

Comment:
I'm not sure if this opens us up to a CSRF attack. I think it doesn't, because this should only affect static files, but does that sound correct?

http://stackoverflow.com/questions/19322973/security-implications-of-adding-all-domains-to-cors-access-control-allow-origin/20186782#20186782